code/nss-patches/license.txt

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016-2017 Microsoft Corporation
```

Comment:
This should be at the very least `INRIA and Microsoft Corporation`

Reply:
It's just copied from your branch ;) I'll change it.